pact-support.gemspec

```diff
@@ -1,39 +1,42 @@
-# -*- encoding: utf-8 -*-
-lib = File.expand_path('../lib', __FILE__)
+# frozen_string_literal: true
+
+lib = File.expand_path("lib", __dir__)
 $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
-require 'pact/support/version'
+require "pact/support/version"
+
+Gem::Specification.new do |spec|
+  spec.name          = "pact-support"
+  spec.version       = Pact::Support::VERSION
+  spec.authors       = ["James Fraser", "Sergei Matheson", "Brent Snook", "Ronald Holshausen", "Beth Skurrie"]
+  spec.email         = ["james.fraser@alumni.swinburne.edu", "sergei.matheson@gmail.com", "brent@fuglylogic.com", "uglyog@gmail.com", "bskurrie@dius.com.au"]
 
-Gem::Specification.new do |gem|
-  gem.name          = "pact-support"
-  gem.version       = Pact::Support::VERSION
-  gem.authors       = ["James Fraser", "Sergei Matheson", "Brent Snook", "Ronald Holshausen", "Beth Skurrie"]
-  gem.email         = ["james.fraser@alumni.swinburne.edu", "sergei.matheson@gmail.com", "brent@fuglylogic.com", "uglyog@gmail.com", "bskurrie@dius.com.au"]
-  gem.summary       = %q{Shared code for Pact gems}
-  gem.homepage      = "https://github.com/bethesque/pact-support"
+  spec.summary       = "Shared code for Pact gems"
+  spec.homepage      = "https://github.com/pact-foundation/pact-support"
+  spec.license       = "MIT"
 
-  gem.required_ruby_version = '>= 2.0'
+  spec.required_ruby_version = ">= 2.0"
 
-  gem.files         = `git ls-files`.split($/)
-  gem.executables   = gem.files.grep(%r{^bin/}).map{ |f| File.basename(f) }
-  gem.test_files    = gem.files.grep(%r{^(test|spec|features)/})
-  gem.require_paths = ["lib"]
-  gem.license       = 'MIT'
+  spec.files         = Dir.chdir(File.expand_path(__dir__)) do
+    `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
+  end
+  spec.executables   = spec.files.grep(%r{^bin/}) { |f| File.basename(f) }
+  spec.require_paths = ["lib"]
 
-  gem.add_runtime_dependency 'randexp', '~> 0.1.7'
-  gem.add_runtime_dependency 'rspec', '>=2.14'
-  gem.add_runtime_dependency 'json'
-  gem.add_runtime_dependency 'term-ansicolor', '~> 1.0'
-  gem.add_runtime_dependency 'find_a_port', '~> 1.0.1'
-  gem.add_runtime_dependency 'thor'
-  gem.add_runtime_dependency 'awesome_print', '~> 1.1'
+  spec.add_runtime_dependency "randexp", "~> 0.1.7"
+  spec.add_runtime_dependency "rspec", ">= 2.14"
+  spec.add_runtime_dependency "json"
+  spec.add_runtime_dependency "term-ansicolor", "~> 1.0"
+  spec.add_runtime_dependency "find_a_port", "~> 1.0.1"
+  spec.add_runtime_dependency "thor"
```

Comment:
Do we use `thor`? I seems not able to find any references from our code.

Reply:
Not from this gem. It can be removed.